### ISSUE TITLE
SUG 9.5 suggests a degenerate model

### ISSUE THREAD

**Issue by jsocolar** [stan-dev/docs#456]
In section 9.5, the full Bayes for the correlated topic model uses a mvnormal variate transformed by a `softmax` as probabilities.  The problem is that the inverse softmax doesn't work as a link function because it isn't unique. My impression from a brief perusal of [Aitchison & Shen 1980](https://www.jstor.org/stable/2335470) is that the solution is to fix one element of the link-scale parameter vector to one, which identifies which unique inverse-softmax we're dealing with as our link function.

The model of [Blei & Lafferty](https://arxiv.org/pdf/0708.3601.pdf) (also cited in the SUG) appears to circumvent the problem by using the maximum a posteriori estimate for the link-scale mvnormal.  Although the posterior is degenerate, it still has a maximum thanks to the prior, and once the estimate is fixed to that maximum the degeneracy disappears.

**Comment by jsocolar** [comment]
Just to note that in the new structure of the SUG, this issue is in the "correlated topic model" subsection of the "clustering models" section of the example models.  

The exact same nonidentifiability issue in the softmax is adequately addressed under the "identifiability" paragraph heading of the "multi-logit regression" subsection of the "regression models" section of the example models; perhaps we could borrow some text there or just link back to that text in the correlated topic model section.
